darasimiolaifa/amistad
# feat(screams): Create and get screams

## What does this PR do?
This PR implements the creation and fetching of screams

## What major activities were carried out?
- set up a firebase application online
- create a local firebase application using firebase tools
- install express to help with writing function endpoints
- create endpoints for creating and getting screams

## How can this be manually tested?
- Clone the repository
- Fetch the branch
- Install the dependencies
- Start project by running `firebase serve` in the terminal
- Copy the localhost url, append `/screams` to it, paste it in postman and send a `post` request to the endpoint using the schema: 
```json
{
    "body": "String",
    "userHandle": "String"
}
```
You should get a successful response.
